rpms/microshift.yml

```diff
@@ -31,3 +31,7 @@ owners:
 # This is used by microshift_sync job
 rhel_targets:
 - 9
+external_scanners:
+  sast_scanning:
+    jira_integration:
+      enabled: true
```

Comment:
Can you add newline at the end of each file?

Reply:
ohh is that a requirement? Didn't know that it was enforced

Reply:
added in https://github.com/openshift-eng/ocp-build-data/pull/3973/commits/af13310cb49e39d5850d39ff0024e03fe04cb30d

Reply:
Yeah it's something of a convention 
https://thoughtbot.com/blog/no-newline-at-end-of-file to setup your editor to automatically follow this